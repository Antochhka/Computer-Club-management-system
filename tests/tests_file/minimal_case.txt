1
00:00 23:59
1
00:01 1 client1
00:02 2 client1 1
00:03 4 client1