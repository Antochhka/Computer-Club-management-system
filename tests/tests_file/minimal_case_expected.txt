00:00
00:01 1 client1
00:02 2 client1 1
00:03 4 client1
23:59
1 1 00:01